guides/v2.1/security/security-overview.md

```diff
@@ -0,0 +1,20 @@
+---
+group: security
+title: Security
+functional_areas:
+  - Configuration
+---
+
+We have added security functionality for Magento Open Source, Commerce, and Commerce (Cloud).  These modules provide enhanced access security to protect your Magento instance by reducing unauthorized access and significantly reducing spam.
```

Comment:
These modules enhance access security to protect your Magento instance from unauthorized access, which can significantly reduce spam.

Reply:
Revised further and updated other content on page